lib/index.js

```diff
@@ -3,12 +3,17 @@ const estraverse = require('estraverse')
 const genCode = require('escodegen')
 const genId = require('./gen-id')
 
-module.exports = function(contents) {
+const sourceMap = require('source-map')
+const SourceMapConsumer = sourceMap.SourceMapConsumer
+
+module.exports = function(contents, sourcemap) {
   const id = genId(this.resourcePath)
   let hasComponent = false
 
   const ast = acorn.parse(contents, {
-    sourceType: 'module'
+    sourceType: 'module',
```

Comment:
I also think I can remove locations and sourceFile option when this.sourceMap is falsy.